### PR TITLE
DOCS-4245, DOCS-1870 Updates Getting Started with Private Locations and Private Locations with Podman

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do?

Adds additional language about creating a Private Location with or without a virtual VM and Podman in the Getting Started with Private Locations and Private Locations documentation.

### Motivation

DOCS-4245, DOCS-1870

### Additional Notes

The Link Checker will fail on the link in the blue banner until this PR is merged.

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/getting_started/synthetics/private_location.md [hunk 1]
```diff
@@ -64,6 +67,8 @@ Your private locations test results display identically to your managed location
     ```
     
     This command starts a Docker container and prepares your private location to run tests. Datadog recommends running the container in detached mode with proper restart policy.
+
+    <div class="alert alert-info">You can use another container runtime such as Podman. For more information, see the <a href="https://docs.datadoghq.com/synthetics/private_location?tab=podman#install-your-private-location">Private Locations documentation</a>.</div>
```
Comment (peculiarism) [inline content/en/getting_started/synthetics/private_location.md:71]: Link isn't working :(

File: content/en/synthetics/private_locations/_index.md [hunk 2]
```diff
@@ -211,6 +211,24 @@ This command starts a Docker container and makes your private location ready to
...
+
+{{% tab "Podman" %}}
+
+The Podman configuration is very similar to Docker, however, you must set `NET_RAW` as an additional capability to support ICMP tests. 
+
+1. Run `sysctl -w net.ipv4.ping_group_range = 0 2147483647` from the host where the container runs.
+2. Run this command to boot your private location worker by mounting your configuration file to the container. Ensure that your `<MY_WORKER_CONFIG_FILE_NAME>.json` file is accessible to mount to the container:
+   
+   ```go
+   podman run –cap-add=NET_RAW --rm -v $PWD/<MY_WORKER_CONFIG_FILE_NAME>.json:/etc/datadog/synthetics-check-runner.json datadog/synthetics-private-location-worker:latest
```
Comment (peculiarism) [inline content/en/synthetics/private_locations/_index.md:224]: @m-rousse could I just get a quick sanity check (main concern is whether you can run the `cap-add` flad during the run command)?

**Review by peculiarism — CHANGES_REQUESTED** [review]
Just one small issue but looks good to me!

File: content/en/getting_started/synthetics/private_location.md [hunk 3]
```diff
@@ -42,7 +42,10 @@ Your private locations test results display identically to your managed location
 
 ## Create your private location
 
-1. Install [Docker][4] on a machine. To get started quickly, you can install Docker on a virtual machine such as [Vagrant Ubuntu 16.04][2].
+1. You can use any Unix virtual machine with Docker or without a virtual machine. Install [Docker][4] on a machine, or use another container runtime such as [Podman][10].
+
+   To get started quickly, you can install Docker on a virtual machine such as [Vagrant Ubuntu 16.04][2].
```
Comment (m-rousse) [inline content/en/getting_started/synthetics/private_location.md:47]: I may have read too fast, though I think the `[2]` link points to the Synthetics PL docker image and not to an Ubuntu 16.04 vagrant box. That would be `https://app.vagrantup.com/ubuntu/boxes/xenial64`

Also, the version 16.04 had its support end in April 2021, could you suggest using 22.04 (`https://app.vagrantup.com/ubuntu/boxes/jammy64`)?

File: content/en/synthetics/private_locations/_index.md (hunk 2 above)
Comment (m-rousse) [inline content/en/synthetics/private_locations/_index.md:224]:
```suggestion
   ```shell
   podman run --cap-add=NET_RAW --rm -it -v $PWD/<MY_WORKER_CONFIG_FILE_NAME>.json:/etc/datadog/synthetics-check-runner.json gcr.io/datadoghq/synthetics-private-location-worker:latest
```

File: content/en/synthetics/private_locations/_index.md [hunk 4]
```diff
@@ -211,6 +211,24 @@ This command starts a Docker container and makes your private location ready to
...
+   ```
+
+   If you have blocked reserved IP addresses, add the `NET_ADMIN` Linux capabilities to your private location container.
```
Comment (m-rousse) [inline content/en/synthetics/private_locations/_index.md:227]:
```suggestion
   If you have configured blocked reserved IP addresses, add the `NET_ADMIN` Linux capabilities to your private location container.
```

File: content/en/synthetics/private_locations/_index.md (hunk 2 above)
Comment (m-rousse) [reply content/en/synthetics/private_locations/_index.md:224]: Yes it works! Updated the command a little

File: content/en/getting_started/synthetics/private_location.md (hunk 1 above)
Comment (alai97) [reply content/en/getting_started/synthetics/private_location.md:71]: This link won't work until we merge this PR in!

File: content/en/getting_started/synthetics/private_location.md [hunk 5]
```diff
@@ -42,7 +42,10 @@ Your private locations test results display identically to your managed location
 
 ## Create your private location
 
-1. Install [Docker][4] on a machine. To get started quickly, you can install Docker on a virtual machine such as [Vagrant Ubuntu 16.04][2].
+1. You can use any Unix virtual machine with Docker or without a virtual machine. Install [Docker][4] on a machine, or use another container runtime such as [Podman][10].
```
Comment (hestonhoffman) [inline content/en/getting_started/synthetics/private_location.md:45]:
```suggestion
1. Install [Docker][4] on a Unix-like machine, or use another container runtime such as [Podman][10].
```
I don't understand the "or without a virtual machine" bit, so this suggestion might change the meaning of the sentence.

**Review by hestonhoffman — COMMENTED** [review]
One small edit

File: content/en/getting_started/synthetics/private_location.md (hunk 5 above)
Comment (peculiarism) [reply content/en/getting_started/synthetics/private_location.md:45]: @hestonhoffman I think this makes sense! I feel documentation wise nothing could beat the Puppet documentation 😉

**Review by m-rousse — APPROVED** [review]

**Pull request merged**